numpy/core/src/umath/override.c

```diff
@@ -180,18 +179,6 @@ normalize_reduceat_args(PyUFuncObject *ufunc, PyObject *args,
 }
 
 static int
-normalize_outer_args(PyUFuncObject *ufunc, PyObject *args,
-                    PyObject **normal_args, PyObject **normal_kwds)
-{
```

Comment:
Never mind the never mind...  in `ufunc_object`, `accumulate` happily accepts `keepdims`. This seems like a mistake.

Anyway, I'm clearly too tired for this, and better stop now.